### PR TITLE
[pledges] update pledge dashboard metatags

### LINKED ISSUE #616: Update pledge metatags to be more search friendly
# Current situation
top result links to this https://www.klimadao.finance/pledge/0xb2B8c7ea4619327101e591828600f62616ad8e12
image

# Ideally
This should say:
```
Crypto Caverns Pledge | KlimaDAO
Crypto Caverns pledges to Offset 1,009 Carbon Tonnes. Read more about their pledge and view their carbon offset history.
```

### PR DESCRIPTION
**Pull request by 0xAeterno**
## Related Ticket

Resolves #616 

## Checklist

- [x] Building the site with `npm run build-site` works without errors
- [x] Building the app with `npm run build-app` works without errors
- [x] I formatted JS and TS files with running `npm run format-all`

### CODE REVIEW

File: site/components/pages/Pledge/PledgeDashboard/index.tsx [hunk 1]
```diff
@@ -50,24 +50,20 @@ export const PledgeDashboard: NextPage<Props> = (props) => {
...
           id: "pledges.dashboard.head.metaTitle",
-          message: `${pledgeOwnerTitle}'s pledge`,
+          message: `${pledgeOwnerTitle}'s Pledge`,
         })}
         metaDescription={t({
-          id: "shared.head.description",
-          message:
-            "Drive climate action and earn rewards with a carbon-backed digital currency.", // Do we need a better metadescription??
+          id: "pledges.dashboard.head.metaDescription",
+          message: `${pledgeOwnerTitle} pledges to Offset ${pledge.footprint} Carbon Tonnes. View their carbon offset history and read more about their commitment.`,
```
Comment (Atmosfearful) [inline site/components/pages/Pledge/PledgeDashboard/index.tsx:61]: pledge.footprint is an array, need to sum the tonnes

image

**Review by Atmosfearful — CHANGES_REQUESTED** [review]

File: site/components/pages/Pledge/PledgeDashboard/index.tsx (hunk 1 above)
Comment (0xAeterno) [reply site/components/pages/Pledge/PledgeDashboard/index.tsx]: My bad, fixed that one up.
Glad I added the total attribute to sum them all up in the db ;)

File: site/components/pages/Pledge/PledgeDashboard/index.tsx [hunk 2]
```diff
@@ -44,30 +44,28 @@ export const PledgeDashboard: NextPage<Props> = (props) => {
...
+  const currentTotalFootprint =
+    pledge.footprint[pledge.footprint.length - 1].total;
 
   return (
     <PledgeLayout canEditPledge={canEditPledge} toggleEditModal={setShowModal}>
       <PageHead
         title={t({
           id: "pledges.dashboard.head.title",
-          message: "KlimaDAO | Pledges",
+          message: "`${pledgeOwnerTitle}'s Pledge | KlimaDAO`",
```
Comment (0xAeterno) [inline site/components/pages/Pledge/PledgeDashboard/index.tsx:55]:
```suggestion
          message: `${pledgeOwnerTitle}'s Pledge | KlimaDAO`,
```

File: site/components/pages/Pledge/PledgeDashboard/index.tsx (hunk 1 above)
Comment (0xAeterno) [reply site/components/pages/Pledge/PledgeDashboard/index.tsx]: image
For some reason the pledged carbon footprints aren't showing, any idea on why that might be? 🤔

Comment (Atmosfearful) [reply site/components/pages/Pledge/PledgeDashboard/index.tsx]: I'll check out and poke around. I also notice this: 
image

**Review by Atmosfearful — APPROVED** [review]
just had to extract strings again, now its good to go!

**Pull request merged**